sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/DoNotHardcodeCredentials.cs

```diff
@@ -55,7 +55,8 @@ private class VariableDeclarationBannedWordsFinder : CredentialWordsFinderBase<V
             public VariableDeclarationBannedWordsFinder(DoNotHardcodeCredentialsBase analyzer) : base(analyzer) { }
 
             protected override string GetAssignedValue(VariableDeclaratorSyntax syntaxNode) =>
-                syntaxNode.Initializer?.Value.ToString();
+                (syntaxNode.Initializer?.Value as LiteralExpressionSyntax)?.Token.ValueText;
```

Comment:
could use `GetStringValue()` extension method from `VisualBasicSyntaxHelper`
I think it should work like
```suggestion
                syntaxNode.Initializer?.Value.GetStringValue();
```